### ISSUE TITLE
New Staffs and Magic Table

### ISSUE THREAD

**Issue by kenijey** [kenijey/harshencastle#85]
I will make a block model for the magic table.
and i will draw an example gui image for the slots
it will just has different slots on different places on the gui, and some magical particles around the block. thats all

Staff of Thunderbolts
-when right click to ground with it, a lightning will arrive on that block(or there)
-the lightning should damage the arounded mobs (it can damage the player if he does it close to him)
-durability for staff

MORE staffs incoming

**Comment by Wyn-Price** [comment]
Should the magic table use the glass containers? That way the glass containers will actually have a use

**Comment by kenijey** [comment]
yes that makes sense. there should be very senseful great slots on magic table, and a small tiny tank in the table. its way to fill that tank in the table to do some magic, u remember actually additions' coffe maker? just like that, a tank in the block/gui to use. maybe 2 tanks

**Comment by kenijey** [comment]
3x glass container slots on the table.

**Comment by Wyn-Price** [comment]
I think I get what you mean
